builder/azure/arm/config.go

```diff
@@ -303,6 +303,8 @@ func setCloudEnvironment(c *Config) error {
 	switch name {
 	case "CHINA", "CHINACLOUD", "AZURECHINACLOUD":
 		c.cloudEnvironment = &azure.ChinaCloud
+	case "GERMAN", "GERMANCLOUD", "AZUREGERMANCLOUD":
```

Comment:
AzureChinaCloud is XPlat CLI.

We may be too forgiving. I can snap to just Azure<Country/Government>Cloud instead.